docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -1,27 +1,77 @@
+* Use the date and time filter to select a time range that you’re interested in exploring. By default, this filter is set to search through the last 24 hours.
+* View detection alerts generated by a specific rule. To do this, click
+*Manage detection rules* and then click on a rule name in the All rules table. A new page showing the detailed view of the rule displays and alert details are stored in the Alerts table beneath the Detection alert trend histogram.
+* Use the *Stack by* dropdown in the Detection alert trend histogram to select specific parameters for which to visualize the individual counts. For example, if you choose to stack by `signal.rule.name`, the histogram will display the total counts by alert name.
+* Augment the results shown in the Alerts table to include alerts from building blocks or only show alerts from indicator match rules. By default, building block alerts are excluded from the Alerts table. Opting to include them expands the results and displays building block alerts alongside regular detection alerts.
+
+NOTE: When updating alert results to also include building block alerts, the Security app searches the `.siem-signals-<Kibana space>` index for the `signal.rule.building_block_type` field. When looking for alerts created from indicator match rules, the app searches the same index for the `signal.rule.threat_mapping` field.
+
+[role="screenshot"]
+image::images/additional-filters.png[Shows multiple ways to filter information]
+
+[float]
+[[customize-the-alerts-table]]
+=== Customize the Alerts table
+Use the icons in the upper left corner of the Alerts table to customize the columns you want displayed and to view the table in full screen mode.
+
+[role="screenshot"]
+image::images/alert-table-columns-and-size.gif[width=100%][height=100%][Demo that shows how to select the customize display icon and full screen icon]
+
+Click the *Customize Event Renderers* icon to enable event renderers within the Alerts table. When enabled, event renderers show relevant details that provide more context to the event. For example, if you enable the *Flow* Event Renderer, the Alerts table shows relevant details describing the flow of the data between a source and destination. These details could include hosts, ports, protocol, direction, duration, amount transferred, process, and geographic location.
+
+[role="screenshot"]
+image::images/customize-event-renderer.png[Shows the Event Renderer icon, 200]
+
+All event renderers are disabled by default. To switch between event views in the Alerts table, you can enable individual event renderers or click *Enable all*. Closing *Customize Event Renderers* page saves your configurations.
+
+[role="screenshot"]
+image::images/customize-event-renderer-page.png[Shows the Event Renderer page]
+
+[float]
+[[view-alert-details]]
+=== View alert details
+To further inspect an alert, click the *View details* icon from the Alerts table.
```

Comment:
I'm on the fence between "icon" and "button." I tend to think of icon as more of a representation of something, and button as something you can actually click or select.

Reply:
Agree with your definition of both - was just trying to stick to the terminology we're currently using across our docs. :) I'll change to "icon" to "button" in this sentence.